### ISSUE TITLE
Maximum Shutter Angle  45° 3200 Iso Max not work on GoPro10 v1.50.70

### ISSUE THREAD

**Issue by wizardcho** [gopro/labs#642]
https://gopro.github.io/labs/control/maxshut/

When check 45° 3200 Iso Max  and scan code. Camera not change Iso Max to 3200. I must change manual.

**Comment by dnewman-gpsw** [comment]
Yes, there is a bug in HERO10 Labs all ISOs work, e.g. 1600 and 6400 both work, except 3200 can't be selected via a QR code.
